Editor/ProjectSetupWindow.cs

```diff
@@ -881,7 +914,11 @@ void DrawNextButton()
                         text = "Validate";
                     }
 
-                    if (buttonDisabled == false && lastDevKeyResponseCode == 200)
+                    if (buttonDisabled == false && lastDevKeyResponseCode == 200 && isResponseJsonValid == false)
+                    {
+                        buttonDisabled = true;
```

Comment:
Consider moving this to line 906 maybe. How about `buttonDisabled = developerkey == null || developerkey.Length == 0 || !isResponseJsonValid`

Do we need to check for `lastDevKeyResponseCode == 200` if `isResponseJsonValid` is already `false`?

Reply:
@parisacognitive3d